starter-repos/app-template/skaffold/render.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
```

Comment:
One question about the file location, I thought app-template only includes resources specific to the application.
The render.yaml file seems to belong to a shared ci/cd folder.

Since the change is pushed to the `app-delivery` branch, what about replacing `../shared-ci-cd/skaffold/render.yaml`?

Reply:
good point, I don't know the answer. @janetkuo Can we change the shared-ci-cd/ folder? I doubt if that will break other customer use cases and the user guid needs to change as well.

Reply:
Here's the script that copies the `starter-repos/shared-ci-cd` folder in Anthos CI/CD to the `platform-admins/shared-ci-cd` repo in the bootstrapped GitLab instance: https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/master/2_gitlab/create-repos.sh

This script is executed in Cloud Build, see [2_gitlab/cloudbuild.yaml](https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/master/2_gitlab/cloudbuild.yaml)

This `app-template` repo will be then downloaded directly by the users - either through `kpt pkg get` or `git clone`. 

Therefore, the `render.yaml` file here is local to the app repo. If you want to reference it in the `platform-admins` project, you'd need to move it to `starter-repos/shared-ci-cd`. 

@Liujingfang1 @haiyanmeng FYI wrt the workflow

Reply:
This script contains the path for 3 different environments, which is specific for this application. If the script can be changed to be more generic, such as auto listing different kustomization folders under config/envs and renders the image there, then it can be moved to `shared-ci-cd`.

Reply:
the shared-ci-cd relies on different repo structure (a k8s sub-directory). My question is that are the existing use cases (e.g. ./anthos-config-management ./golang-template etc) going to change to use the app-template repo structure or keep their existing setup. If it's the latter, we should not replace the shared-ci-cd.